### PR TITLE
fix bug in passwords.py

### PR DESCRIPTION
**Pull request by bricehalder**
python not c++

### CODE REVIEW

File: wikispeedruns/auth/passwords.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def check_password(user: dict, password: str) -> bool:
     id = user["user_id"]
     hash = user["hash"]
 
-    if (user["is_old_hash"]):
+    if "is_old_hash" in user and user["is_old_hash"]:
```
Comment (dqian3) [inline wikispeedruns/auth/passwords.py:28]: Why would this not be in user? It should always be queried?

**Review by dqian3 — CHANGES_REQUESTED** [review]
W

File: wikispeedruns/auth/passwords.py (hunk 1 above)
Comment (bricehalder) [reply wikispeedruns/auth/passwords.py:28]: Screen Shot 2022-02-24 at 7 31 32 PM

this was the error I was trying to fix - not sure if it's just an issue locally or not.

I was trying to log into a user that was created before the recent password changes

Comment (dqian3) [reply wikispeedruns/auth/passwords.py:28]: can you add the full stack trace?

Comment (bricehalder) [reply wikispeedruns/auth/passwords.py:28]: Screen Shot 2022-02-24 at 9 18 57 PM

Repro:
on recent pull of main branch, no local changes
attempt to login to account on local instance

Comment (bricehalder) [reply wikispeedruns/auth/passwords.py:28]: nvm, solution is to re-run createDB script

**Pull request closed without merging**